.ci_support/linux_64_numpy1.22python3.10.____cpython.yaml

```diff
@@ -21,5 +25,7 @@ python:
 target_platform:
 - linux-64
 zip_keys:
+- - c_stdlib_version
```

Comment:
@jaimergp @isuruf (sorry for the random ping here)

Is this a bug in conda-smithy? the double `-` in `- - c_stdlib_version` looks odd